plasmapy/atomic/atomic.py

```diff
@@ -1229,39 +1096,42 @@ def common_isotopes_for_element(argument, most_common_only):
 
 
 def stable_isotopes(argument=None, unstable_instead=False):
-    """Returns a list of all stable isotopes of an element, or if no input is
-    provided, a list of all such isotopes for every element.
+    """Returns a list of all stable isotopes of an element, or if no
+    input is provided, a list of all such isotopes for every element.
 
     Parameters
     ----------
     argument: integer or string
-        A string or integer representing an atomic number or element, or a
-        string represnting an isotope.
+        A string or integer representing an atomic number or element,
+        or a string represnting an isotope.
 
     unstable_instead: boolean
```

Comment:
Actually, how about just `unstable`?

Reply:
I've been going back and forth about how best to name this.  The possibilities I've been wondering about are:
1. `stable_isotopes('Fe', unstable_instead=True)`  (status quo)
2. `stable_isotopes('Fe', unstable=True)`  (your suggestion)
3. `stable_isotopes('Fe', invert=True)`
4. `unstable_isotopes('Fe')`  (create a new function)
5. Not provide this functionality and let people do a list comprehension like:
```Python
[isotope for isotope in known_isotopes('H') if isotope not in stable_isotopes('H')]
```
Are any of these better than the others?  The most readable is `unstable_isotopes('Fe')`, though I'm wondering if we need to create a function for it.